catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/integration/test/CatalogIcebergBaseIT.java

```diff
@@ -61,93 +60,67 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.apache.commons.lang3.StringUtils;
-import org.apache.hadoop.hdfs.HdfsConfiguration;
 import org.apache.iceberg.NullOrder;
 import org.apache.iceberg.PartitionField;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.SortField;
+import org.apache.iceberg.catalog.SupportsNamespaces;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.exceptions.NoSuchNamespaceException;
-import org.apache.iceberg.hive.HiveCatalog;
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SparkSession;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
-@Tag("gravitino-docker-it")
-public class CatalogIcebergIT extends AbstractIT {
-  public static String metalakeName = GravitinoITUtils.genRandomName("iceberg_it_metalake");
-  public static String catalogName = GravitinoITUtils.genRandomName("iceberg_it_catalog");
-  public static String schemaName = GravitinoITUtils.genRandomName("iceberg_it_schema");
-  public static String tableName = GravitinoITUtils.genRandomName("iceberg_it_table");
-  public static String alertTableName = "alert_table_name";
-  public static String table_comment = "table_comment";
-
-  public static String schema_comment = "schema_comment";
-  public static String ICEBERG_COL_NAME1 = "iceberg_col_name1";
-  public static String ICEBERG_COL_NAME2 = "iceberg_col_name2";
-  public static String ICEBERG_COL_NAME3 = "iceberg_col_name3";
-  public static String ICEBERG_COL_NAME4 = "iceberg_col_name4";
-  private static final String provider = "lakehouse-iceberg";
-
-  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
-  private static String WAREHOUSE;
-  private static String HIVE_METASTORE_URIS;
-
-  private static String SELECT_ALL_TEMPLATE = "SELECT * FROM iceberg.%s";
-  private static String INSERT_BATCH_WITHOUT_PARTITION_TEMPLATE =
-      "INSERT INTO iceberg.%s VALUES %s";
-  private static GravitinoMetalake metalake;
-
-  private static Catalog catalog;
-
-  private static HiveCatalog hiveCatalog;
-
-  private static SparkSession spark;
+public abstract class CatalogIcebergBaseIT extends AbstractIT {
+
+  protected static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+  protected String metalakeName = GravitinoITUtils.genRandomName("iceberg_it_metalake");
```

Comment:
Besides `TYPE` `WAREHOUSE` `URIS`,  the other field no need to make protected?

Reply:
yes, i will fix it.

Reply:
updated.